File: sistema_facturacion_api/sistema_facturacion_api.Useful/OperationResultRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sistema_facturacion_api.Useful
{
    public class OperationResultRequest
    {
        public bool Succcess { get; set; }
        public string Message { get; set; }
        public dynamic Data { get; set; }
        public Pager Paginacion { get; set; }
    }
}

File: sistema-facturacion-front/src/app/models/dtos/operation-result-request.ts
import { Pager } from "./pager";

export class OperationResultRequest{
  succcess: boolean;
  message: string;
  data: any;
  paginacion: Pager | any;
  token: string;
  fechaExpiracion: Date;

  constructor(){
    this.succcess = false;
    this.message = "";
    this.token = "";
    this.fechaExpiracion = new Date();
  }
}
